numpy/lib/tests/test_histograms.py

```diff
@@ -524,6 +524,16 @@ def test_simple_range(self):
                 msg += " with datasize of {0}".format(testlen)
                 assert_equal(len(a), numbins, err_msg=msg)
 
+    def test_integer(self):
+        """
+        Test that bin width for integer data is at least 1.
+        """
+        estimator_list = ['fd', 'scott', 'rice', 'sturges', 'auto']
```

Comment:
looks like a good use case for a parametrized test

Reply:
this follows test_simple_weighted below, if you want to convert all of them to parametrized that would better be done as a separate PR imo.